tests/VMelnalksnis.PaperlessDotNet.Tests.Integration/Documents/DocumentClientTests.cs

```diff
@@ -94,6 +95,21 @@ public async Task Create()
 
 		updatedDocument.Title.Should().Be(update.Title);
 
+		var documentDownload = await Client.Documents.Download(id, original: true);
+
+		documentDownload.ContentType.Should().Be("text/plain");
+		documentDownload.FileName.Should().Contain("Lorem Ipsum");
+
+		using (var stream = documentDownload.Content)
+		{
+			Assert.IsNotNull(stream);
+
+			StreamReader reader = new StreamReader(stream);
+			string text = reader.ReadToEnd();
+
+			Assert.IsTrue(text.Contains("Lorem ipsum"));
```

Comment:
Should use `FluentAssertions` for more helpful error messages when the test fails. Also, can check all of the file contents like this
```suggestion
			var expectedContent = await typeof(DocumentClientTests).ReadResource(documentName);
			text.Should().BeEquivalentTo(expectedContent);
```